torch/csrc/utils/python_arg_parser.h

```diff
@@ -742,6 +761,10 @@ static auto check_has_torch_function(PyObject* obj) -> bool
  *
  */
 
+auto handle_torch_function(PythonArgs &r, PyObject* self, PyObject* args, PyObject* kwargs, PyObject* torch_api, const char* module_name) -> PyObject*;
```

Comment:
What are the differences between these three `handle_torch_function` overloads? In what cases would you call each one?

Reply:
Added comments.